.changeset/four-trains-raise.md

```diff
@@ -0,0 +1,20 @@
+---
+"astro-integration-kit": patch
```

Comment:
It is breaking already because it changes on which hook it can be used.

Reply:
I don't think it's breaking because addDts does not change in itself

Reply:
It received the params of `astro:config:setup`. Now it receives the params of `astro:config:done`. It will break everywhere it's being used and require a code change moving it to the new hook